tests/python/gaia-ui-tests/gaiatest/brick/manifest.ini

```diff
@@ -0,0 +1,8 @@
+[parent:../manifest.ini]
+
+[DEFAULT]
+external = false
+online = true
```

Comment:
It's a minor nit, but I don't think these tests will require [an Internet connection](https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/tests/manifest.ini#L27)